docker/nginx-consul-template/nginx.conf.tmpl

```diff
@@ -0,0 +1,32 @@
+events {
+    worker_connections 4096;
+}
+
+http {
+{{- with $service_query := (printf "%s.%s" (env "DE_ENV") (env "NGINX_PROXY_SERVICE_NAME")) }}
```

Comment:
(I also factored this out into a variable. Because sometimes I can refactor usefully :P